tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py36
+
+[testenv]
+whitelist_externals = rm
+commands=
+    ; Force packaging even if setup.{py,cfg} haven't changed
+    rm -rf ./*.egg-info/
+    coverage run setup.py test
+    pylama tests setup.py kytos
```

Comment:
Sorry! I have made a mistake about the repo, I was thinking about kytos-napps. This is ok!